docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -365,6 +365,14 @@ will be stored.
 For "sqlite3" only. [Shared cache](https://www.sqlite.org/sharedcache.html) setting used for connecting to the database. (private, shared)
 Defaults to `private`.
 
+### query_retries
+
+For "sqlite" only. How many times to retry a query in case of database is locked failures. Default is 0 (disabled).
+
+### transaction_retries
+
+For "sqlite" only. How many times to retry a transaction in case of database is locked failures. Default is 5.
```

Comment:
```suggestion
This setting applies to `sqlite` only and controls the number of times the system retries a transaction when the database is locked. The default value is `5` (disabled).
```

Reply:
@chri2547 5 is not the same as disabled?

Reply:
That looks like my fault. Just removed the word disabled. Thank you!